i3_resurrect/layout.py

```diff
@@ -115,7 +119,7 @@ def restore(workspace, numeric, directory, profile):
             util.eprint(f'Could not find saved layout for profile "{profile}"')
```

Comment:
As I said, this whole `except FileNotFoundError` block (lines 117 to 122) can be deleted as it is handled in `layout.read()` now instead.